### ISSUE TITLE
Label Name

### ISSUE THREAD

**Issue by MentalGear** [aleju/imgaug#392]
I was looking through the Bounding Boxes labeling exmaples, and I noticed that there were no parameter were I could specify a name for each label.
Example: If I have a picture with two animals, I want to keep the labels animalA and animalB, not just that they are 2 different bounding boxes.

Example Code (that doesn't specify a label):
```
bbs = BoundingBoxesOnImage([
    BoundingBox(x1=25, x2=75, y1=25, y2=75),
    BoundingBox(x1=100, x2=150, y1=25, y2=75),
    BoundingBox(x1=175, x2=225, y1=25, y2=75)
], shape=image.shape)
```

**Comment by aleju** [comment]
Not entirely sure if I understand you correctly, but bounding boxes already have a `label` parameter, which can be used e.g. via `BoundingBox(x1=25, x2=75, y1=25, y2=75, label="Person")`. The examples mostly don't mention it yet though. Is that what you want?

**Comment by MentalGear** [comment]
Yes, exactly! Just couldn't find any reference to thre label setting.

**Comment by MentalGear** [comment]
Btw thanks for the excellent work on imgaug !
